hyppo/kgof/base.py

```diff
@@ -0,0 +1,45 @@
+"""
+Module that will contain several goodness-of-fit test methods
+"""
+
+from __future__ import division
+
+from abc import ABC, abstractmethod
+
+
+class GofTest(ABC):
+    """Abstract class for a goodness-of-fit test."""
```

Comment:
A base class for a discriminability test.

Reply:
Changed.